scripts/snakemake_sophie_format_template_experiment_public.R

```diff
@@ -0,0 +1,101 @@
+library(readr)
+library(dplyr)
+library(tibble)
+
+# read in counts and metadata ---------------------------------------------
+
+# set strain using wildcard so that proper set of control samples can be extracted
+strain <- unlist(snakemake@wildcards[['strain']])
+# strain <- "pao1"
+
+# read in hogan lab sample counts
+#counts <- read_csv("outputs/combined_compendia/num_reads_pao1.csv")
+counts <- read_csv(snakemake@input[['counts']], show_col_types = F)
+
+# read in hogan lab sample metadata
+#metadata <- read_csv("inputs/metadata.csv")
+metadata <- read_csv(snakemake@input[['metadata']], show_col_types = F) %>%
+  filter(!experiment_accession %in% c("SRX7101177", "SRX7101178", "SRX7101179", "SRX7101180", "SRX7101181")) # filter out low count samples; see comment below
+
+# below I recorded the counts for these samples when mapped against PAO1.
+# there aren't enough counts so I removed them here.
+# SRX7101177 SRX7101178 SRX7101179 SRX7101180 SRX7101181 
+# 131        476         49        164        263
+# Every other library has at least 7000 reads mapped to pao1.
+
+# determine experiment that is being compared ----------------------------------
+
+# figure out what comparison should be formatted with this run by parsing the 
+# snakemake wildcard. Each "hogan_comparison" contains two experiments, separated
+# by "-vs-".
+comparison <- unlist(snakemake@wildcards[['pub_comparison']])
+# comparison <- 'pub-vs-lbpao1'
+
+# parse comparison to determine which two sample sets to place in template files
+set1 <- gsub("-.*", "", comparison)
+set2 <- gsub(".*-vs-", "", comparison)
+
+# separate hogan lab samples into sets based on sample type --------------------
+
+pub_samples <- metadata %>%
+  filter(pa_in_reads == TRUE) 
+
+m63_samples <- c("M631", "M632", "M633")
+
+lbpao1_samples <- c('SRX5661364', 'SRX5661363', 'SRX6976948', 'SRX6976949',
```

Comment:
Are these liable to change at all, if this is run in the future with more samples?

Reply:
no -- these are a hand-curated set of controls. someone could add more to them, but I think the set I came up with is comprehensive enough that it shouldn't need to change even if additional experimental samples were added